### DISCUSSION TITLE
/zh-cn/blog/posts/2023_summary

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#136] [Announcements]
# 2023 年度小结 - Reverier's Blog

👴不是刚写完2022的年度小结🐴.jpg 写的时候回过头看了看2022的年度小结，感觉时间过的是真快啊。想想今年可能是我这辈子最有意思的一年，

https://blog.woooo.tech/posts/2023_summary/

**Comment by spn21** [comment]
希望神在工作之余更注重下身体😭😭😭

**Reply by Reverier-Xu** [reply]
活着就是胜利（？）
